src/gui/widgets/FxLine.cpp

```diff
@@ -208,6 +208,7 @@ void FxLine::contextMenuEvent( QContextMenuEvent * )
 	contextMenu->addHelpAction();
 	contextMenu->exec( QCursor::pos() );
 	delete contextMenu;
+	delete this;
```

Comment:
Oh yeah,  guess you're right. Thanks for catching this one.

However how can I be able to deallocate this object on the move left/right case?